orderer/common/cluster/deliver.go

```diff
@@ -100,7 +106,11 @@ func (p *BlockPuller) PullBlock(seq uint64) *common.Block {
 			p.Logger.Errorf("Failed pulling block [%d]: retry count exhausted(%d)", seq, p.MaxPullBlockRetries)
 			return nil
 		}
-		time.Sleep(p.RetryTimeout)
+
+		if waitOnStop(p.RetryTimeout, p.StopChannel) {
+			p.Logger.Warn("Received a stop signal")
```

Comment:
why is it warning? it's fine to stop it. i think it should be info

Reply:
done